Update readme with latest MSI package. Rename Actions workflow.

aaronwaggener opened the pull request:
This is me very gently kicking the can down the road. I believe we can have a better workflow for creating the MSI installer and I also believe it will be vastly simplified when we no longer need to natively build extensions for `node serial-ports`. I'd like to revisit all of this once the WebUSB work is complete and we slim this app down to be a very simple Chromium based browser.

For now, this is just an update to the Readme to point to the latest MSI installer I built and some updated instructions. I also renamed the Github Actions file to be consistent with the ml-activities repo.

madelynkasula reviewed (commented):
you may already have this on your list, but josh has a zendesk macro to help users who request an MSI: https://codeorg.zendesk.com/agent/admin/macros/360023503152

we could obviously update the macro every time we publish a new version, but maybe we should move this documentation to this repo, include a step in the README about updating that documentation, and have the macro link to it instead? i suppose we could add the step to update the macro in this README, but i think it'd be easier to keep all documentation in the same place

aaronwaggener commented:
@madelynkasula Apologies for the delayed response here. I did a little research and tracked down four places where we were linking directly to the .msi download
1. [ZenDesk macro](https://codeorg.zendesk.com/agent/admin/macros/360023503152) 
1. ZenDesk support page [Installing the Code.org Maker App to a Group of Computers](https://support.code.org/hc/en-us/articles/360029870131-Installing-the-Code-org-Maker-App-to-a-Group-of-Computers) 
1. ZenDesk support page [Maker Toolkit and Circuit Playground Support](https://support.code.org/hc/en-us/articles/115003407851-Maker-Toolkit-and-Circuit-Playground-Support) 
1. [GitHub Readme](https://github.com/code-dot-org/browser#silent-install--group-install) 

It would be easier for us to have everyone reference our README for those instructions but I feel like ultimately Zendesk is the best place for any user support information to live. As such, **I updated everything to reference item 2 above.** I also added a note in the MSI release instructions to update the link in Zendesk after each release. 

Long term/ideally the link in Zendesk would point to something like `maker-latest.msi` which would redirect to our latest build and we would have a step in a release script to update the redirect on AWS. Given the low churn in this repo, I think the manual update is ok for now but definitely worth considering further automation if we see an uptick in releases.

madelynkasula reviewed (approved):
that sounds good -- i hadn't realized we already had a zendesk article! thank you for deduplicating the MSI instructions, too
